libs/langchain/langchain/vectorstores/rocksetdb.py

```diff
@@ -303,15 +304,15 @@ def _build_query_sql(
         where_str = f"WHERE {where_str}\n" if where_str else ""
         return f"""\
 SELECT * EXCEPT({self._embedding_key}), {distance_str}
-FROM {self._collection_name}
+FROM {self._workspace}.{self._collection_name}
```

Comment:
this isn't backwards compatible right? are we ok with that? (will let you make call)

Reply:
yes, it's not backwards compatible, but it should be finewe don't have any users of our langchain client yet. Rockset is a SaaS product, so we can be sure of that :)